src/azure-iot-edge-installer.sh

```diff
@@ -33,8 +33,15 @@ chmod +x install-container-management.sh
 chmod +x install-edge-runtime.sh
 chmod +x validate-post-install.sh
 
+# check if current OS is Tier 1
+. /etc/os-release
+is_os_tier1 $ID $VERSION_ID
+if [ "$?" != "0" ]
+then 
+    log_error "This OS is not supported. Exit."
```

Comment:
nit: should we leave a link to the Tier 1 OS documentation, or show that only two OS are supported today?